src/user-guide/writing-workflows/external-triggers.rst

```diff
@@ -107,8 +160,7 @@ the :cylc:conf:`[xtriggers]` section:
        [[foo]]
            script = run-foo.sh
 
-However, when xtriggers are declared the name used must adhere to the following
-rules:
+However, xtrigger names must adhere to the following rules:
```

Comment:
"However" made me re-read the last example (starting at "A zero-offset..") but in reality, the two are unrelated. Consider removing the "However".

Perhaps consider putting this in a `..note::` directive?

Reply:
Huh, yeah, I suspect that line got repositioned at some point. It should also be in the general section, not clock triggers.